packages/tizen-remote/test/e2e/server.js

```diff
@@ -0,0 +1,136 @@
+import {promisify} from 'node:util';
+import {WebSocketServer} from 'ws';
+import {parse as parseUrl} from 'url';
+import d from 'debug';
+import delay from 'delay';
+
+const debug = d('tizen-remote:test:e2e:server');
+
+/**
+ * @type {WeakMap<import('ws').WebSocket,boolean>}
+ */
+const heartbeats = new WeakMap();
+
+const KEEP_ALIVE_TIMEOUT = 1000;
+
+/**
+ * Websocket server for E2E API testing
+ */
+export class TestWSServer extends WebSocketServer {
```

Comment:
wow you made a mock server

Reply:
I wanted to get going on it before I learned to talk to the TV properly 😄